### ISSUE TITLE
Page

### ISSUE THREAD

**Issue by matheusps** [vtex/shoreline#2036]
### Problem

There are designs where the background is soft instead of base. This is not supported by the Page component by default. 

### Solution

Accept a variant on the page

```jsx
<Page variant="soft | base">
```

### References

Figma Page: https://www.figma.com/design/xmWRVrjShWlZwchJg0huzZ/Product-%E2%80%93-FastStore-WebOps-v2?node-id=929-75490&node-type=frame&m=dev.

**Comment by beatrizmilhomem** [comment]
Agreed!

Related issue to define the pattern https://github.com/vtex/shoreline/issues/1788

**Comment by renatamottam** [comment]
Nice! This background is being used on dashboards proposals. It'll definitely help :)

**Comment by lucasaarcoverde** [comment]
> One token is called bg-base and the other is bg-base-soft. Won't it be confusing for the prop values to be named soft and base?

I agree with that! Maybe we should take the `primary` and `secondary` approaches, which would open space for other variants

**Comment by felipepowlist** [comment]
Isn't it a matter of the header having a background color vs being transparent? 🤔
